conf/application.conf

```diff
@@ -172,6 +172,13 @@ mongo-async-driver {
   }
 }
 
+today-date-for-validation-flag {
+    is-2021 = false,
```

Comment:
also , cant we figure out this flag based on the below date, instead of declaring a separate flag ? 
may `.getYear` tells you whether its 2021 or not ?

Reply:
yes, sure i could but now i'll need to change few things , not sure is worth it given will all be gone in 2 weeks - let me know I'll change it if really bothers you